frontend/src/components/TransactionLinks/TransactionLink.vue

```diff
@@ -99,6 +103,22 @@ export default {
           this.toastError(this.$t('gdd_per_link.not-copied'))
         })
     },
+    copyLinkWithText() {
+      navigator.clipboard
+        .writeText(
+          `${this.link}\n` +
+          `${this.$store.state.firstName} ${this.$t('transaction-link.send_you')} ${this.amount} Gradido.\n` +
+          `\"${this.memo}\"\n` +
+          `${this.$t('gdd_per_link.credit-your-gradido')} ${this.$t('gdd_per_link.validUntilDate', {date: this.$d(new Date(this.validUntil), 'short')})}`
```

Comment:
```suggestion
           `${this.link}\n
            ${this.$store.state.firstName} ${this.$t('transaction-link.send_you')} ${
            this.amount
          }. GDD \n
            "${this.memo}"\n
            ${this.$t('gdd_per_link.credit-your-gradido')} ${this.$t(
            'gdd_per_link.validUntilDate',
            { date: this.$d(new Date(this.validUntil), 'short') },
          )}`
```

Reply:
the pluses are superfluous. a single apostrophe at the beginning and end is sufficient.

Reply:
We solved this in another suggestion.